### ISSUE TITLE
`paasta logs --components chronos` fails with TypeError: cannot use a string pattern on a bytes-like object

### ISSUE THREAD

**Issue by EvanKrall** [Yelp/paasta#1413]
probably need to stick a `b` in front of a regex pattern literal somewhere.

```
$ paasta logs -s REDACTED -c REDACTED -i REDACTED --components chronos
Fetching the last 100 lines and applying filters...
Traceback (most recent call last):
  File "/usr/bin/paasta", line 11, in <module>
    load_entry_point('paasta-tools==0.66.13', 'console_scripts', 'paasta')()
  File "/opt/venvs/paasta-tools/lib/python3.6/site-packages/paasta_tools/cli/cli.py", line 122, in main
    return_code = args.command(args)
  File "/opt/venvs/paasta-tools/lib/python3.6/site-packages/paasta_tools/cli/cmds/logs.py", line 1106, in paasta_logs
    return pick_default_log_mode(args, log_reader, service, levels, components, clusters, instances)
  File "/opt/venvs/paasta-tools/lib/python3.6/site-packages/paasta_tools/cli/cmds/logs.py", line 1033, in pick_default_log_mode
    raw_mode=args.raw_mode,
  File "/opt/venvs/paasta-tools/lib/python3.6/site-packages/paasta_tools/cli/cmds/logs.py", line 771, in print_last_n_logs
    self.run_code_over_scribe_envs(clusters=clusters, components=components, callback=callback)
  File "/opt/venvs/paasta-tools/lib/python3.6/site-packages/paasta_tools/cli/cmds/logs.py", line 566, in run_code_over_scribe_envs
    callback(component, stream_info, scribe_env, cluster=cluster)
  File "/opt/venvs/paasta-tools/lib/python3.6/site-packages/paasta_tools/cli/cmds/logs.py", line 768, in callback
    parser_fn=stream_info.parse_fn,
  File "/opt/venvs/paasta-tools/lib/python3.6/site-packages/paasta_tools/cli/cmds/logs.py", line 786, in filter_and_aggregate_scribe_logs
    line = parser_fn(line, clusters, service)
  File "/opt/venvs/paasta-tools/lib/python3.6/site-packages/paasta_tools/cli/cmds/logs.py", line 304, in parse_chronos_log_line
    utc_timestamp = extract_utc_timestamp_from_log_line(line)
  File "/opt/venvs/paasta-tools/lib/python3.6/site-packages/paasta_tools/cli/cmds/logs.py", line 276, in extract_utc_timestamp_from_log_line
    tokens = re.match(iso_re, line)
  File "/opt/venvs/paasta-tools/lib/python3.6/re.py", line 172, in match
    return _compile(pattern, flags).match(string)
TypeError: cannot use a string pattern on a bytes-like object
```

cc @cblkwell

**Comment by ronin13** [comment]
This fails for --components marathon too.

**Comment by ddelnano** [comment]
I started something in #1439

**Comment by ddelnano** [comment]
This has been merged in #1464
